examples/hal/Cargo.toml

```diff
@@ -25,7 +25,9 @@ env_logger = "0.5"
 image = "0.18"
 log = "0.4"
 winit = "0.13"
+glsl-to-spirv = "0.1.4"
```

Comment:
this reminds me: we have custom glsl-to-spirv support in some of the backends, like Vulkan.
It doesn't make any sense now, since all the backends accept SPIR-V, so any integration with glsl should be done outside of the backends.

Reply:
If I understand correctly, you want to strip out glsl-to-spirv from the backends? I think that's outside this PR's job.

Reply:
Yes, hence the part "this reminds me" :) No action expected from this PR